Makefile

```diff
@@ -82,5 +82,11 @@ lint:
 
 fmt:
 	gofmt -s -l -w $(SRCS)
+
+pull:
```

Comment:
On second thought: can we call this target `pull-images`?

`pull` seems a bit too generic / unspecific to me.